src/lib/classes/styling/GlobalCssSchemesLoader.ts

```diff
@@ -0,0 +1,206 @@
+import type ColorValue from "./ZodSchemas/AttributeSchemas/ColorAttribute";
+import type MediaScheme from "./ZodSchemas/MediaScheme";
+
+import MediaSchemes from "./ZodSchemas/MediaSchemes";
+import GlobalCssProperties from "../../GlobalCssProperties.json";
+
+import type { z } from "zod";
+
+/**
+ * Class for handling the loading of different properties based on active media features
+ */
+class GlobalCssSchemesLoader {
+	private _window: Window;
+	private _mediaSchemes: z.infer<typeof MediaSchemes>;
```

Comment:
nitpick: why are you adding these as private members? You can define them as a `const` outside of the class and still use them in the class. For `window` you don't need to define it at all, just reference it directly :)